### PR TITLE
[Doc] Add ADDITIONAL_SERVICE in real time integration

### PR DESCRIPTION
**Pull request by mbounabi**
https://jira.kisio.org/browse/NAVP-1149

### CODE REVIEW

File: documentation/slate/source/includes/realtime.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ The effect of a disruption can be one of the following:
 	<li>[SIGNIFICANT_DELAYS](#SIGNIFICANT_DELAYS)</li>
 	<li>[REDUCED_SERVICE](#REDUCED_SERVICE)</li>
 	<li>[MODIFIED_SERVICE](#MODIFIED_SERVICE)</li>
+    <li>[ADDITIONAL_SERVICE](#ADDITIONAL_SERVICE)</li>
```
Comment (pbougue) [inline documentation/slate/source/includes/realtime.md:15]: minor: weird indent

**Review by pbougue — APPROVED** [review]

File: documentation/slate/source/includes/realtime.md (hunk 1 above)
Comment (mbounabi) [reply documentation/slate/source/includes/realtime.md:15]: Fixed. Though it didn't change anything once built

File: documentation/slate/source/includes/realtime.md [hunk 2]
```diff
@@ -356,3 +358,104 @@ At the added stop area, the departure time of the train with a modified service
...
+            ]
+            "to": ⊕{5 items},
+            "type": "public_transport",
+        },
+       ⊕{10 items}
+    ]
+```
+
+The status of the journey is `ADDITIONAL_SERVICE`. This new journey can only be displayed if "data_freshness" is set to "realtime".  
+A list of the disruptions impacting the journey is also present at the root level of the response.
```
Comment (nonifier) [inline documentation/slate/source/includes/realtime.md:453]:
```suggestion
A list of disruptions impacting the journey is also present at the root level of the response.  
```

File: documentation/slate/source/includes/realtime.md [hunk 3]
```diff
@@ -356,3 +358,104 @@ At the added stop area, the departure time of the train with a modified service
...
+    ]
+```
+
+The status of the journey is `ADDITIONAL_SERVICE`. This new journey can only be displayed if "data_freshness" is set to "realtime".  
+A list of the disruptions impacting the journey is also present at the root level of the response.  
+A link to the concerned disruption can be found in the section "display_informations".  
+
+<div></div>
+### Departures & Stop Schedules
+At one of the added stop area from the additional trip, the departure time of the train added is displayed if "data_freshness" is set to "realtime".
```
Comment (nonifier) [inline documentation/slate/source/includes/realtime.md:458]:
```suggestion
At one of the added stop area from the additional trip, the departure time of the added train is displayed if "data_freshness" is set to "realtime".  
```

**Review by nonifier — APPROVED** [review]

**Review by kadhikari — APPROVED** [review]

File: documentation/slate/source/includes/realtime.md (hunk 2 above)
Comment (mbounabi) [reply documentation/slate/source/includes/realtime.md:453]: Done in an other PR

File: documentation/slate/source/includes/realtime.md (hunk 3 above)
Comment (mbounabi) [reply documentation/slate/source/includes/realtime.md:458]: Done in an other PR

**Pull request merged**